### DISCUSSION TITLE
Todo and some points to be discussed

### DISCUSSION THREAD

**Discussion by TheFatBlue** [DFY-NCSU/slash-phase6#25] [General]
Hi, guys. I just summarize some points we need to do based on the Rubric for Repo. 

**TODO**:
- doc
- two video, a short version for advertising and a longer one for showing our work
- add workflow to run test cases automatedly
- add `INSTALL.md`
- update `README.md`, ensuring contains the following content:
    - a video link
    - badges for many many things
    - doi
- update `CONTRIBUTING.md`

Besides, there's some points I'm not so sure for the requirements, which needs further discussion before being settle down:
- version control tool: should it be git? but we're using **Git**hub, for sure we use it
- evidence that the whole team using the same tool: what kind of tools? does it means we all can run locally?
- evidence that the whole team work across multiple places in the code base: emmmm, im confused...

looking forward to your reply!

**Comment by TheFatBlue** [comment]
@TomasAndersonFang @IMYXR

**Reply by TomasAndersonFang** [reply]
@TheFatBlue Thanks for your notes, and I will discuss with you soon~

**Comment by TheFatBlue** [comment]
@TomasAndersonFang ru at hunt? I’d like to discuss them with you

**Comment by IMYXR** [comment]
Seem Good，I can also join the talk after 3:30(due to the TA MEETING)!

**Comment by TheFatBlue** [comment]
I've reserved the room 3243 from 3:00 PM to 5:00 PM, we can meet there first
